### PR TITLE
DOCS-3509 Integrating Monitors with Statuspage Guide

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds Integrating Monitors with Statuspage guide with some screenshots.

### Motivation

DOCS-3509

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by urseberry** [comment]
DOCS-3904

**Review by peculiarism — APPROVED** [review]

File: content/en/monitors/guide/integrate-monitors-with-statuspage.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Integrating Monitors With Statuspage
```
Comment (hestonhoffman) [inline content/en/monitors/guide/integrate-monitors-with-statuspage.md:2]:
```suggestion
title: Integrating Monitors with Statuspage
```

Comment (hestonhoffman) [reply content/en/monitors/guide/integrate-monitors-with-statuspage.md:2]: No clear guidance on this, but it looks like we don't capitalize prepositions or articles in our titles.

**Review by hestonhoffman — APPROVED** [review]
Sorry for the delay. One minor style suggestion, but this is great!

**Pull request merged**